packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -24,6 +25,7 @@ const VectorRendering = ({
   const [renderTypeOptions, setRenderTypeOptions] = useState<string[]>([
     'Single Symbol',
   ]);
+  const [SymbologyTab, setSymbologyTab] = useState<SymbologyTab>('color');
```

Comment:
```suggestion
  const [symbologyTab, setSymbologyTab] = useState<SymbologyTab>('color');
```

Woops, looks like a search-and-replace caused us to shadow the type name :)

Reply:
482f458